src/components/request/useRequestTableData.tsx

```diff
@@ -249,8 +253,38 @@ function useRequestTableData({
       ],
     });
 
+    if (rules !== undefined) {
```

Comment:
fyi this might also be needed in `src/components/request/useSettledTableData.tsx`. This is the table which shows on the page once the request is settled

Reply:
Just to confirm, the objective of adding it to `src/components/request/useSettledTableData.tsx` would be to also display the rules in this table? Right now the rules are still displayed in the Input Data for settled proposals but I can also put the rules in this table if you think it would be helpful?

![image](https://user-images.githubusercontent.com/76416300/195417034-bb7ba11f-b741-4927-8d85-d4fb2459ead7.png)

Reply:
oh thats great, then no further work is needed there